common-arcana.lic

```diff
@@ -518,16 +518,17 @@ module DRCA
     end
   end
 
-  def update_astral_data(data)
+  def update_astral_data(data, settings)
```

Comment:
💡  As a new argument, this is a breaking  change to other scripts that may use these methods and aren't currently passing in the settings. I suggest providing a default `settings = OpenStruct.new` in each of the updated method signatures.
```suggestion
  def update_astral_data(data, settings = OpenStruct.new)
```